src/index.js

```diff
@@ -0,0 +1,167 @@
+const state = {
+    degrees: 67,
+    city: 'Seattle'
+}
+
+const apiCallLocation = () => {
+    const endpointLocation = "http://127.0.0.1:5000/location"
+    const endpointWeather = "http://127.0.0.1:5000/weather"
```

Comment:
LOVE that you pulled out these URLs to the top of the function! Since we won't change these variables, we can even make them into constants!